toffy/json_utils_test.py

```diff
@@ -77,3 +77,28 @@ def test_list_moly_fovs(tmpdir):
     pred_moly_fovs = json_utils.list_moly_fovs(tmpdir)
 
     assert np.array_equal(pred_moly_fovs.sort(), moly_fovs.sort())
+
+
+def test_read_json_file():
```

Comment:
You should use a temporary directory here, otherwise the files get written to your actual computer. The temporary directory that gets created is treated like a new base directory. Then you can add folders, files, etc inside like normal

https://github.com/angelolab/toffy/blob/main/toffy/file_hash_test.py#L12